src/views/ContentPage.vue

```diff
@@ -2,20 +2,21 @@
   <div>
     View our latest {{ data.type }}
     <ContentList :type="data.type" />
+    <pagination :type="data.type" :limit="12" />
```

Comment:
does this even work, shouldn't it be `limit="12"` considering `:limit=""` with the preceding colon infers that the value should be a defined variable?

Reply:
I can confirm this does not work. The `loadItems()` call is being triggered with the default `limit` the first time it runs. If I set the default `limit` param to say 6 and then have my `:limit` set to 12, it will return 6 the first time it runs and then return 12 on every page after that. I need to figure out a way to do this better.